dts/bindings/fuel-gauge/maxim,max17048.yaml

```
@@ -8,3 +9,22 @@ description: |
 compatible: "maxim,max17048"
 
 include: [i2c-device.yaml, fuel-gauge.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+  overvoltage-threshold:
+    description: |
+      Overvoltage in mV. If MAX17048_TRIGGER is set, an interrupt is triggered upon
```

Don't reference Kconfigs in the devicetree binding.
```suggestion
      Overvoltage in mV. If triggers are enabled, an interrupt is triggered upon
```